### DISCUSSION TITLE
bigfix/11.0/mcm/MCM/Install/c_installation_guide

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#23] [Q&A]
# bigfix/11.0/mcm/MCM/Install/c_installation_guide

Read this guide to learn about the requirements and available installation scenarios to ensure that the deployment of BigFix MCM and BigFix Mobile goes smoothly in your environment.

https://help.hcltechsw.com/bigfix/11.0/mcm/MCM/Install/c_installation_guide.html

**Comment by gayathripg** [comment]
test
